src/jslib.js

```diff
@@ -368,6 +368,13 @@ let jslib = {
 		let u = checkjs(L, 1);
 		lua.lua_pushnumber(L, +u);
 		return 1;
+	},
+	"instanceof": function(L) {
+		let u1 = checkjs(L, 1);
```

Comment:
Hrm. maybe `tojs` isn't perfect either: I think we don't want the fallthrough/default case.